atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/TimestampCommand.java

```diff
@@ -31,17 +35,35 @@
     		description = "Get the current immutable timestamp")
     private boolean immutable;
     
+    @Option(name = {"-d", "--date-time"},
+            description = "Return the earliest real point in time at which the immutable timestamp" +
```

Comment:
I was confused by 'real point in time' and would prefer 'wallclock time' but idk if that's just me